k8s/example/values.yaml

```diff
@@ -84,10 +84,10 @@ global:
     releaseTag: v1
     # A download-able url for an "decrypted" agent binary that will be used as
     # the connectivity runtime in the target environment
-    binaryURL: https://github.com/EC-Release/sdk/releases/agent
+    # binaryURL: https://github.com/EC-Release/sdk/releases/agent
```

Comment:
binaryURL is currently in production usage for v213. Suggest to uncomment and leave as-is.